src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -18,30 +19,31 @@
 public static class Program
 {
 
-    public static async Task Main(string[] args)
+    public static async Task<int> Main(string[] args)
     {
-        if (!TryParseCli(args, out Configuration config))
+        if (!TryParseCli(args, out Configuration? config))
         {
-            return;
+            return 2;
```

Comment:
What is the significance of '2'?  Would '3' be better? It might be nice to define some constants with useful names to help readers understand this code better.

Reply:
That's a good idea.